src/Fantomas.Tests/TriviaTests.fs

```diff
@@ -541,3 +542,33 @@ let x =
     | [ { Type = TriviaNodeType.MainNode SynConst_Unit
           ContentBefore = [ Directive "#if DEBUG\n\n#endif" ] } ] -> pass ()
     | _ -> Assert.Fail(sprintf "Unexpected trivia: %A" trivia)
+
+[<Test>]
+let ``double try-with, comment before inner 'with' not duplicated, 1969`` () =
+    // This test fails because of [Expected: "\r\ntr" But was:  "\ntry\n"] at start position of the string
```

Comment:
@nojaf  Do you have any hints why this test might not pass here at 'TriviaTests.fs' but identical passes in 'CompilerDirectivesTests.fs?' Are there any different configurations for different test files?

Reply:
Not entirely sure this will help but please format all your changed files.
Run `dotnet fantomas src -r`, you will need to do this anyway in order to pass the CI build.

Reply:
Yes I'll do that as soon as possible 👍

Reply:
Thanks